migration/idempotent/016-vacuum-engine.sql

```diff
@@ -0,0 +1,40 @@
+DO $block$
+BEGIN
+    IF _prom_catalog.is_timescaledb_installed() THEN
+        CREATE OR REPLACE VIEW _ps_catalog.chunks_to_vacuum_freeze AS
+        SELECT
+            c.hypertable_id,
+            c.id as chunk_id,
+            cc.id as compressed_chunk_id,
+            cc.schema_name,
+            cc.table_name,
+            t.vacuum_count,
+            t.autovacuum_count,
+            t.last_vacuum,
+            t.last_autovacuum,
+            k.relallvisible
+        FROM _timescaledb_catalog.chunk c
+        INNER JOIN _timescaledb_catalog.chunk cc ON (c.dropped = false and c.compressed_chunk_id = cc.id)
+        INNER JOIN
+        (
+            SELECT
```

Comment:
`INNER JOIN _timescaledb_catalog.dimension d ON (d.id = ds.dimension_id)` seems to be sufficient in my metric queries. It's the only one we have.

Reply:
Agreed. However, a user may create hypertables of their own and define more than one dimension on them. This view does not return ONLY metric chunks, but chunks tied to any hypertable